Samples/WMCConsole_winrtcpp/README.md

```diff
@@ -0,0 +1,44 @@
+
+<!---
+  samplefwlink: TBD
+--->
+
+# Win32 Desktop sample applications for using the [Wiindows Media Capture APIs](https://docs.microsoft.com/en-us/uwp/api/Windows.Media.Capture.MediaCapture)
+
+	This folder contains sample application that demonstrate the Windows Media Capture API usage patterns for using the camera related features in Win32 Desktop applications for Windows 10. 
```

Comment:
>sample application that demonstrate [](start = 22, length = 35)

either "**a** sample application that demonstrate**s**" or "sample application**s** that demonstrate"